google-services.json

```diff
@@ -0,0 +1,40 @@
+{
+  "project_info": {
+    "project_number": "203972072487",
+    "firebase_url": "https://disciple-tools-app.firebaseio.com",
+    "project_id": "disciple-tools-app",
+    "storage_bucket": "disciple-tools-app.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:203972072487:android:35cef86b74095ccafd2a02",
+        "android_client_info": {
+          "package_name": "tools.disciple.app"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "203972072487-ifhknov95kigura8dts3111f7h4pv3b2.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyAU36b9_TgHfZjaR089wDRykQQjaUBUXbk"
```

Comment:
It's probably time that we as a Team come to a consensus on how we want to deal with API keys in the source.  There are a few different options, but we don't want to store them openly.  Any suggestions?  One common route is to use a .env or config.yml or whatever and then add it to .gitignore, but that potentially causes problems for us if we want to continue Continuous Delivery to Expo.

Reply:
We definitely don't want all of the keys in this file exposed. Normally I use environment variables which are easy to configure in CI tools, but I haven't had to deal with this google-services.json file before. My quick searches don't provide a super clear path, so it will require a bit more research. I'd be curious if there is a way to make this a dummy file and then make use of environment variables to override those dummy values.